src/behavior.rs

```diff
@@ -59,50 +91,108 @@ pub trait Behavior<Pane> {
     #[allow(clippy::fn_params_excessive_bools)]
     fn tab_ui(
         &mut self,
-        tiles: &Tiles<Pane>,
+        tiles: &mut Tiles<Pane>,
         ui: &mut Ui,
         id: Id,
         tile_id: TileId,
-        active: bool,
-        is_being_dragged: bool,
+        state: TabState,
     ) -> Response {
         let text = self.tab_title_for_tile(tiles, tile_id);
+        let close_btn_size = Vec2::splat(ui.spacing().icon_width);
```

Comment:
You use `icon_width` but then call `.shrink(2.0)` on it later on, so it isn't really `icon_width`.

Just make it configureable with `Behavior::close_button_size` instead